netpyne_ui/netpyne_geppetto.py

```diff
@@ -64,6 +68,7 @@ def __init__(self):
         logging.debug("Initializing the original model")
 
         jupyter_geppetto.context = {'netpyne_geppetto': self}
+        self.loaded_mods=set()
```

Comment:
This instance variable doesn't seem use in the rest of the PR, is it required?

Reply:
@filippomc

Reply:
Probably a leftover from some attempt to avoid crashes due to modules loaded twice. Thanks for finding out!